cumulusci/tasks/bulkdata/tests/test_snowfakery.py

```diff
@@ -676,39 +770,6 @@ def test_running_totals_repr(self):
     #             mapping = yaml.safe_load(f)
     #     return mapping
```

Comment:
Why are there commented tests here?

Reply:
They are old tests of generate_and_load_from_yaml. Over time they get ressurrected as snowfakery tests. You can kind of see that process in this PR. Commented out tests are updated and become "real" tests. It serves both as:

1. A todo list of features that have not yet achieved parity with `generate_and_load_from_yaml`.
2. A todo list of features that might exist but are not fully tested yet.
3. An archive of patterns of how to test them.

It will go away over time.